packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -216,8 +216,8 @@ export function createLodestarMetrics(
 
     // Finalized block and proposal stats
     allValidators: {
-      expected: register.gauge({
-        name: "lodestar_all_validators_expected_count",
+      total: register.gauge({
+        name: "lodestar_all_validators_total_count",
```

Comment:
non-intuitive metric name, might wanna refactor now or later

```suggestion
        name: "lodestar_all_validators_proposals_total_count",
```

or

```suggestion
        name: "lodestar_all_validators_blocks_total_count",
```

Reply:
hmmm... i think expected was more intutive than total for me ... but anyway whatever makes things more intuitive to the users. :+1:

Reply:
As per discussion in discord it was not clear what is meant by "expected", user might think it means "schedulded" aka lookahead of upcoming blocks.

Not sure if total is the correct term, see my other comment, maybe I am misunderstanding it